manifests/g/Guru3D/Afterburner/4.6.5.Beta4/Guru3D.Afterburner.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Guru3D.Afterburner
+PackageVersion: 4.6.5.Beta4
+PackageLocale: en-US
+Publisher: MSI
+PublisherUrl: https://www.msi.com/Landing/afterburner/graphics-cards
```

Comment:
This doesn't seem right, since the download on that page has a different SHA

Reply:
That's because archives are different (Guru3D one has a folder about their site). Executables inside them are the same, though